### ISSUE TITLE
Can't wait for this

### ISSUE THREAD

**Issue by johntdyer** [briangann/grafana-sensu-datasource#1]
![omg so excited](http://3.bp.blogspot.com/-IySnx19-_Qw/UnRWkdjkXLI/AAAAAAAAGnY/5y8OTLBHsg8/s1600/3r3paq.jpg)

**Comment by aviramartac** [comment]
Any plans to support graphs? or only data tables?

**Comment by johntdyer** [comment]
Any plans to submit as an official data source?

**Comment by briangann** [comment]
Yes this will be submitted - though it really needs to be made into an app for grafana to include more features that uchiwa allows "out of the box"

**Comment by briangann** [comment]
This does support graphs - you can also use singlestat and other plugins like the vonage status panel to show counts of events (warning/crits) and more.

Making this an app would make it easier to use with pre-made dashboards and custom panels.
